dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/createfolder/CreateFolderPresenter.java

```diff
@@ -113,7 +115,17 @@ public void createFolder() {
         }));
         createFolderTask.setOnCancelled(SafeHandler.logHandle(event -> this.closeDialog()));
         createFolderTask.setOnFailed(SafeHandler.logHandle(event -> {
-            alert.error(CREATE_FOLDER_ERR_LOGS);
+            final Throwable e = event.getSource().getException();
+            if (e instanceof FailedRequestException) {
+                alert.error("folderAlreadyExists");
```

Comment:
We should verify that this is for sure a conflict, and if not, then we need to display a different error and log it.